### ISSUE TITLE
Error: Webpack Compilation Error (Module parse failed: Unexpected token) (optional chaining operator)

### ISSUE THREAD

**Issue by yaobao1993** [archfz/cypress-terminal-report#254]
Hello, I am trying to use cypress-terminal-report 6.1.2 (the latest version).

When Cypress runs with this new plugin, it prints this:
```
Error: Webpack Compilation Error
./node_modules/cypress-terminal-report/src/collector/LogCollectSimpleControl.js 44:48
Module parse failed: Unexpected token (44:48)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|     }
| 
>     if (testState === 'failed' && mochaRunnable?._retries > 0) {
|       testTitle += ` (Attempt ${mochaRunnable?._currentRetry + 1})`
|     }
 @ ./node_modules/cypress-terminal-report/src/installLogsCollector.js 18:32-78
 @ ./cypress/support/e2e.ts
```

After some investigation, I found the issue might be the "optional chaining operator", and it is introduced from this commit (https://github.com/archfz/cypress-terminal-report/commit/ea03e6946df6fa63c0491c3448398bf48454199b).

I switched cypress-terminal-report to version 6.0.2 (without above commit), and it works.

Is "optional chaining operator" valid in this project? Or might the user (Cypress user) change some configuration to overcome this? I can use version 6.0.2, but perhaps the latest version can address this issue as well.

Thanks

**Comment by archfz** [comment]
This was semi reported already and PR was submitted https://github.com/archfz/cypress-terminal-report/pull/250

I am waiting for @wemcdonald to fix the PR and I will do a release ASAP. If he doesn't respond I will replicate the changes.

**Comment by archfz** [comment]
Released in 6.1.3
